zstacklib/zstacklib/utils/lvm.py

```
@@ -1627,12 +1631,62 @@ def _deactive():
+def discard_lv(path, raise_exception=True):
+    with OperateLv(path):
+        if lv_in_use(path):
+            err = "lv discard failed because %s in use" % path
+            if raise_exception:
+                raise Exception(err)
+
+            logger.warn(err)
+            return
+
+        try:
+            length = long(get_lv_size(path))
```

Ruff (0.12.2)

Undefined name `long`

(F821)